### PR TITLE
Create default topSites for new tab page

### LINKED ISSUE #5594: newtab - Have default top sites available
When there aren't any top sites yet (brand new session), we can pre-load Brave with a few top sites.

cc: @bradleyrichter 

[twitter (@brave)(pinned)](https://twitter.com/brave)
[facebook (brave)](https://www.facebook.com/BraveSoftware/)
[Brave.com](https://brave.com/)
[github (brave)](https://github.com/brave/browser-laptop)
youtube (brave)
playstore (brave)

### PR DESCRIPTION
**Pull request by cezaraugusto**
- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).

Fix #5594

Auditors: @bsclifton

@bradleyrichter:
* I edited topSites icon default size, from 32px to 64px. It affects not only default topSites but any other site. Can be changed as needed;
* Default sites' icon are now stored within Brave. I used arbitrary icons, can also be changed as needed.

Test Plan:
1) On a fresh (clean session-store) view, new tab should have a default grid with 6 sites,
being them described on referenced issue
2) Visiting a new site should override previous grid
3) If session-store is previously populated, default topSites shouldn't appear on grid

Note: Since we're having some issues with accessed sites, it may be required to refresh the new site in order for (2) to work

### CODE REVIEW

**Comment by bbondy** [comment]
This needs to be rebased please

**Comment by bbondy** [comment]
@bradleyrichter did we decide on these top sites? 
I would put the iOS app instead of youtube

**Comment by bbondy** [comment]
If we are going with these I think we should cache these tile icons as well so we don't send requests to these places without a user visiting them.  @bradleyrichter ?

**Comment by bradleyrichter** [comment]
@bbondy and we could use larger favicons so the initial view is crispy...

**Comment by bradleyrichter** [comment]
I think youtube is going to get more traffic than facebook but I will ask Catherine.

**Comment by bradleyrichter** [comment]
Catherine suggested to swap Git for iOS app store, and this matches BE's opinion.

**Comment by bsclifton** [comment]
Change looks good- only concern I have (@bbondy, @bradleyrichter) is this.  We could do it two ways:

1. Only show the default sites if you have 0 sites.  As soon as you have 1, they all disappear.
2. Show the 6 default sites and then have the user's sites take priority over them. ex: if the user has only 2 top sites showing, grab 4 of the default ones.

This PR is doing option 1... is that what we'd like? I personally would prefer option 2, but I'm curious what you all think :smile: 

Besides that, all that's left is the feedback from @bbondy; we can likely put a high res copy of the favicon into [app/extensions/brave/img/](https://github.com/brave/browser-laptop/tree/master/app/extensions/brave/img)

Here's what it looks like right now; notice the default sites have a larger favicon size (64px x 64px):
screen shot 2016-11-16 at 5 19 25 pm

**Comment by bradleyrichter** [comment]
please change to #2.

And when we have freceny done, the defaults will stay around longer and not be replaced even before a user sees them for the second time

@garvankeeley may have comments related to his improvements over the same issues with iOS top sites.

**Comment by bbondy** [comment]
Still needs to load icons not remotely, moving this to 0.12.11

**Comment by cezaraugusto** [comment]
PR updated.

@bbondy sites' icon are now stored within the app;
@bradleyrichter I updated this [PR resume](https://github.com/brave/browser-laptop/pull/5623#issue-189272120) with info about design changes I did. All open for new changes per request.
@bsclifton code ready for review

File: js/data/newTabData.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const fs = require('fs')
```
Comment (bsclifton) [inline js/data/newTabData.js]: This is never used, can you remove it pls? :smile:

**Comment by cezaraugusto** [comment]
@bsclifton thanks, updated

**Comment by bsclifton** [comment]
Looks good to me :smile:

**Pull request merged**